### PR TITLE
Create AccessToken entity, AccessTokenGenerator service

### LINKED ISSUE #205: Create `AccessTokenGenerator` service
### Description
Create `AccessToken` entity, `AccessTokenGenerator` service

### Requirements
- project should be runnable

### Deliverables
- `AccessToken` entity
-  `JwtSettings` external configuration model and configuration values
- `ClaimConstants` user claims
- package installed for `Application` project
- `AccessTokenConfiguration` entity configuration
- `AccessTokenRepository` repository
- `AccessTokenService` foundation service
- fluent validation
- `AccessTokenGeneratorService` helper service
- add migration
- register all necessary infrastructure to DI

### PR DESCRIPTION
**Pull request by Jasurbek1223**
Closes #205

### CODE REVIEW

**Review by botirjonabdumannonovv — DISMISSED** [review]

**Review by botirjonabdumannonovv — APPROVED** [review]

File: src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Common/Validators/AccessTokenValidator.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        RuleSet(
+            EntityEvent.OnCreate.ToString(),
+            () =>
+            {
+                // Ensure that IsRevoked is not set to true during creation.
+                RuleFor(accessToken => accessToken.IsRevoked).NotEqual(true);
+
+                // Ensure that UserId is not empty during creation.
+                RuleFor(accessToken => accessToken.UserId).NotEqual(Guid.Empty);
+            }
```
Comment (sulton-max) [inline src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Common/Validators/AccessTokenValidator.cs:32]: Shouldn't we check `Token` and `ExpiryTime` properties too ?

**Review by sulton-max — CHANGES_REQUESTED** [review]

File: src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Common/Validators/AccessTokenValidator.cs (hunk 1 above)
Comment (Jasurbek1223) [reply src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Common/Validators/AccessTokenValidator.cs]: It has been changed

**Comment by Jasurbek1223** [comment]
Just check the Access Token Validator section

**Review by sulton-max — APPROVED** [review]

**Review by sulton-max — APPROVED** [review]
Good job

**Review by rashidovasadbek — APPROVED** [review]

File: src/Training.AirBnb.Clone.Backend/AirBnB.Domain/Entities/AccessToken.cs
[the remark this answers is not in the document]
Comment (sulton-max) [reply src/Training.AirBnb.Clone.Backend/AirBnB.Domain/Entities/AccessToken.cs:1]: yes, which files didn't have comment ?

**Pull request merged**